proto/frequenz/api/microgrid/common.proto

```diff
@@ -73,19 +74,21 @@ message MetricAggregation {
 // Metrics of a DC electrical connection.
 message DC {
   // The DC voltage across the component.
-  // In Volts (V).
+  // In Volt (V).
```

Comment:
I understand that this is being done for consistency, but in text or in speech, plural units are more natural for some metrics. E.g., we say 230 volts and 5 amperes, and not 220 volt and 5 ampere.

Reply:
It is for consistency and because it's specifying the unit (Volt, Ampere, Watt, etc.), is not part of a phrase speaking about a quantity, like 220 Volts. For me it sounds more correct to specify units in singular, but if you think it's clearer to specify units in plural, I can change it, no problem.